packages/vscode-extension/src/accountTree.ts

```diff
@@ -369,3 +488,28 @@ async function setSubscription(subscription: SubscriptionInfo | undefined) {
     ]);
   }
 }
+
+async function getSideloadingStatus(token: string): Promise<boolean | undefined> {
+  const instance = axios.create({
+    baseURL: "https://dev.teams.microsoft.com",
+    timeout: 30000,
+  });
+  instance.defaults.headers.common["Authorization"] = `Bearer ${token}`;
+  try {
+    const response = await instance.get("/api/usersettings/mtUserAppPolicy");
+    let result: boolean | undefined;
+    if (response.status >= 400) {
+      result = undefined;
+    } else {
+      result = response.data?.value?.isSideloadingAllowed as boolean;
+    }
+
+    tools.telemetryReporter?.sendTelemetryEvent(TelemetryEvent.CheckSideloading, {
+      [TelemetryProperty.IsSideloadingAllowed]: result + "",
+    });
+    return result;
+  } catch (error) {
+    tools.telemetryReporter?.sendTelemetryErrorEvent(TelemetryEvent.CheckSideloading, error);
+    return undefined;
```

Comment:
Is it necessary to retry when exception happens?

Reply:
Good point. Added retry logic.